ComponentKit/DataSources/Common/CKComponentDataSourceAttachController.mm

```diff
@@ -11,7 +11,6 @@
 #import <objc/runtime.h>
 #import "CKComponentInternal.h"
 
-#import "CKComponentDataSourceAttachController.h"
```

Comment:
Makes sense! I can undo that. It's a personal habit (assuming internal headers import regular headers)